cache/lru_cache.cc

```diff
@@ -464,14 +470,15 @@ void LRUCacheShard::Promote(LRUHandle* e) {
 
       // Insert a dummy handle into the primary cache. This dummy handle is
       // not IsSecondaryCacheCompatible().
+      // FIXME? This should not overwrite an existing non-dummy entry in the
+      // rare case that one exists
       Cache::Priority priority =
           e->IsHighPri() ? Cache::Priority::HIGH : Cache::Priority::LOW;
-      s = Insert(e->key(), e->hash, /*value=*/nullptr, 0,
+      s = Insert(e->key(), e->hash, kDummyValueMarker, /*charge=*/0,
```

Comment:
kDummyValueMarker is 18 characters long, so the charge should be 18; otherwise, this becomes a hidden memory usage.

Reply:
`kDummyValueMarker[...]` is static data. There is no incremental hidden memory usage.

Reply:
(aside from the pre-existing limitation of not accounting for metadata used by dummy handles)